### PR TITLE
add visualisation.py

### PR DESCRIPTION
**Pull request by Damir21**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "cgshop2025-visualization"
+version = "0.1.0"
+description = "A script for visualizing CGSHOP2025 data"
+authors = ["Damir Salakhov <damir-salakhov@mail.ru>"]
+license = "MIT"
```
Comment (nbykh) [inline pyproject.toml:6]: our license is GNU 3.0, but it doesn't matter much haha

**Review by nbykh — APPROVED** [review]

**Comment by nbykh** [comment]
nice job!

File: scripts/visualisation.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import os
+import json
+import matplotlib.pyplot as plt
+
+
+def save_plot(fig, filename):
+    base_name = os.path.basename(filename).replace('.instance.json', '')
+    output_dir = 'visualisations'
```
[the remark this answers is not in the document]
Comment (Damir21) [reply scripts/visualisation.py:8]: Added

File: pyproject.toml (hunk 1 above)
Comment (Damir21) [reply pyproject.toml]: Changed

**Pull request merged**